Gemfile

```diff
@@ -9,7 +9,7 @@ gemspec
 SOURCE         = ENV.fetch('SOURCE', :git).to_sym
 REPO_POSTFIX   = SOURCE == :path ? ''                                : '.git'
 DATAMAPPER     = SOURCE == :path ? Pathname(__FILE__).dirname.parent : 'https://github.com/datamapper'
-DM_VERSION     = "~> #{DataMapper::VERSION}"
```

Comment:
Why not consider adding another constant `DataMapper::MIN_VERSION`, just so you don't have to touch the Gemfile each time a version needs to be softly bumped?